CHANGELOG.md

```diff
@@ -9,14 +9,14 @@ Removed legacy code and dependencies.
 ### New
```

Comment:
Does it worth mentioning the support of configuring snippet user in #899 ?

Reply:
I think it's fine.